Modules/IO/MRC/src/itkMRCHeaderObject.cxx

```diff
@@ -110,52 +110,6 @@ MRCHeaderObject::SetHeader(const Header * buffer)
   this->m_ExtendedHeader = nullptr;
   this->m_ExtendedFeiHeader = nullptr;
 
-  SizeValueType extendedHeaderBytes = 0;
-  if (this->m_Header.nreal & 1)
-  {
-    extendedHeaderBytes += 2;
-  }
-  if (this->m_Header.nreal & 2)
-  {
-    extendedHeaderBytes += 6;
-  }
-  if (this->m_Header.nreal & 4)
-  {
-    extendedHeaderBytes += 3;
-  }
-  if (this->m_Header.nreal & 8)
-  {
-    extendedHeaderBytes += 2;
-  }
-  if (this->m_Header.nreal & 16)
-  {
-    extendedHeaderBytes += 2;
-  }
-  if (this->m_Header.nreal & 32)
-  {
-    extendedHeaderBytes += 4;
-  }
-  if (this->m_Header.nreal & 64)
-  {
-    extendedHeaderBytes += 2;
-  }
-  if (this->m_Header.nreal & 128)
-  {
-    extendedHeaderBytes += 4;
-  }
-  if (this->m_Header.nreal & 256)
-  {
-    extendedHeaderBytes += 2;
-  }
-  if (this->m_Header.nreal & 512)
-  {
-    extendedHeaderBytes += 4;
-  }
-  if (this->m_Header.nreal & 1024)
-  {
-    extendedHeaderBytes += 2;
-  }
```

Comment:
How many tests invoke this code? If just one, maybe figure out what `extendedHeaderBytes` should be and compare against it here? Just removing code does not feel right in this instance.

Reply:
I threw in a null dereference and ran all tests.  The following hit the removed code:

```
	929 - itkMRCImageIOTest (ILLEGAL)
	930 - itkMRCImageIOTest2a (ILLEGAL)
	931 - itkMRCImageIOTest2b (ILLEGAL)
	932 - itkMRCImageIOTest2c (ILLEGAL)
```

Looking at git blame, the code I removed has been dead code for a very long time.

I agree just removing it is not ideal without understanding this code.  According to `git log` every commit for about a decade has been just stylistic-type stuff, so I'm not sure anyone actually knows this code. :)

Reply:
Looks like these classes were originally contributed by @blowekamp and @dave3d , so maybe they still have some data that would require these code (?) or know the purpose of the relevant part.

Reply:
@seanm I have encountered the below warning..... Look like I missed an e-mail 🤣